arch/xtensa/src/esp32s3/esp32s3_lcd.c

```diff
@@ -300,13 +310,16 @@ static const struct fb_videoinfo_s g_base_videoinfo =
 
 /* This structure provides the base layer interface */
 
-static const struct fb_vtable_s g_base_vtable =
+static struct fb_vtable_s g_base_vtable =
```

Comment:
The `fb_remove_paninfo(&g_base_vtable, FB_NO_OVERLAY);` I added takes a `FAR struct fb_vtable_s *vtable`. Looks like `fb_remove_paninfo` does not modify it, though. Maybe the parameter could be made const? See https://github.com/apache/nuttx/blob/52b0df1ce6cdcc99b44700b20a2bc3351285eee0/drivers/video/fb.c#L1607

Otherwise, should I keep the vtable const in esp32s3_lcd.c and cast to non-`const`?

Reply:
Ok, let's keep as it.

Reply:
I understand why it has to be non-`const` now. There's a member `priv` at the end of the struct that gets assigned by fb.c. In `master` the `const` region is being written to.